charts/hawkbit/templates/tests/test-connection.yaml

```diff
@@ -11,8 +11,12 @@ metadata:
     "helm.sh/hook": test-success
 spec:
   containers:
-    - name: wget
-      image: busybox
-      command: ['wget']
-      args:  ['{{ include "hawkbit.fullname" . }}:{{ .Values.service.port }}']
+    - name: curl
+      image: curlimages/curl
+      command: ['curl']
+      args: [
+        "-X", "GET",
+        "-u", "admin:admin",
```

Comment:
I think instead of `admin` as username/password here, corresponding `Values` references should be put in here (excluding the `{noop}` password prefix).